sdk/storage/azure-storage-files-datalake/src/datalake_directory_client.cpp

```diff
@@ -281,6 +281,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
         item.ExpiresOn = _detail::Win32FileTimeConverter::Win32FileTimeToDateTime(
             std::stoll(path.ExpiresOn.Value()));
       }
+      item.EncryptionContext = std::move(path.EncryptionContext);
```

Comment:
what is this? Is this a bug fix?

Reply:
Yeah, it's a bug fix.

Reply:
If it's a bug fix, let's open a separate PR, add testing and update changelog.

Reply:
Will use this PR as a bug fix PR and open a spearate PR for other test cases.